packages/notifi-react/README.md

```diff
@@ -1,81 +1,139 @@
+import { NotifiContextProvider, NotifiCardModal } from '@notifi-network/notifi-react';
+import { useEthers } from '@usedapp/core';
+import { providers } from 'ethers';
+
+const NotifiCard = () => {
+
+  const { account, library } = useEthers();
+  const signer = useMemo(() => {
+    if (library instanceof providers.JsonRpcProvider) {
+      return library.getSigner();
+    }
+    return undefined;
+  }, [library]);
+
+  if (account === undefined || signer === undefined) {
+    // account is required
+    return null;
+  }
 
-const App = () => {
-  const params = {
-    // params for contexts
-  };
   return (
-    <NotifiContextProvider {...params}>
-      <App />
-    </NotifiContextProvider>
+//tenantId/dAppId and cardId are found on the Notifi Tenant Portal. Please see Docs below.
+ <NotifiContextProvider
+    tenantId="YOUR_TENANT_ID"
+    env="Production"
+    cardId="YOUR_CARD_ID"
+    signMessage={signMessage}
+    walletBlockchain="ETHEREUM"
+    walletPublicKey={account}
+    >   
+        <NotifiCardModal darkMode={true} />
+  </NotifiContextProvider>
   );
 };
 ```
+> [Docs](https://docs.notifi.network/docs/getting-started)
+
+<br/><br/>
+
+# Advanced Customized Implementation
+
+
 
-2. Mount `NotifiCardModal` component to wherever you want in your dapp.
+The style of the `NotifiCardModal` is fully customizable. There are two approaches to customize the card style:
+<br/><br/>
 
+## Customizing the Card Styling
+
+#### **Global CSS override (*Recommended*)**
+ By overriding the default CSS classes, we can style the elements in the card.
+   [Default CSS Classes are found here.](https://github.com/notifi-network/notifi-sdk-ts/tree/stage-notifi-react-m2/packages/notifi-react/lib/style)
+
+*Example*
 ```tsx
-import { NotifiCardModal } from '@notifi-network/notifi-react';
+import { NotifiContextProvider, NotifiCardModal } from '@notifi-network/notifi-react';
+import { useEthers } from '@usedapp/core';
+import { providers } from 'ethers';
+import "../../../styles/notifi/index.css";
+
+const NotifiCard = () => {
+
+  const { account, library } = useEthers();
+  const signer = useMemo(() => {
+    if (library instanceof providers.JsonRpcProvider) {
+      return library.getSigner();
+    }
+    return undefined;
+  }, [library]);
+
+  if (account === undefined || signer === undefined) {
+    // account is required
+    return null;
+  }
 
-const MyComponent = () => {
-  const cardModalParams = {
-    // params for NotifiCardModal
-  };
-  return <NotifiCardModal {...cardModalParams} />;
+  return (
+//tenantId/dAppId and cardId are found on the Notifi Tenant Portal. Please see Docs below.
+ <NotifiContextProvider
+    tenantId="YOUR_TENANT_ID"
+    env="Production"
+    cardId="YOUR_CARD_ID"
+    signMessage={signMessage}
+    walletBlockchain="ETHEREUM"
+    walletPublicKey={account}
+    >   
+        <NotifiCardModal darkMode={true} />
+  </NotifiContextProvider>
+  );
 };
 ```
 
```

Comment:
1. I think this section should not be located under `Global CSS override` section. 
2. Do we have to put the particular implimentation for `signer` with `@usedapp/core`?
     - This lib looks not super common
     - The impl method vary according to chain and lib. So shall we just do like 
     - I do not see where is the impled `signMessage`. And do we want to add a sub-section to describe how to impl `signMessage` if needed
I would do
```tsx
// ... Necessary import statements
const NotifiCard = () => {
  
  const walletPublicKey = // grab the wallet public key from connected wallet
  const signMessage = // implement the corresponding sign message method according to your preferred blockchain

  
  return (
//tenantId/dAppId and cardId are found on the Notifi Tenant Portal. Please see Docs below.
 <NotifiContextProvider
    tenantId="YOUR_TENANT_ID"
    env="Production"
    cardId="YOUR_CARD_ID"
    signMessage={signMessage}
    walletBlockchain="ETHEREUM"
    walletPublicKey={walletPublicKey}
    >   
        <NotifiCardModal darkMode={true} />
  </NotifiContextProvider>
  );
};

Reply:
I had it to highlight the import - but ill refactor to this.

Reply:
For the quick start ill still use this. The implementation should be as quick as copy pasting into a react lib to get started